setup.md

```diff
@@ -33,9 +33,7 @@ Wed 21 Apr 2021 11:38:19 BST
 ## Git Version Control Tool
 Git is a program that can be accessed from your command line tool.
 
-- On Windows, it is recommended to use **Git Bash**, which comes included as part of the [Git For Windows package](https://gitforwindows.org/) and will
-  install the Bash command line tool as well as Git.
-  - *For AstraZeneca-managed computers (where you may not have admin permissions), you can obtain Git For Windows package from the AstraZeneca Software Store.*
+- On Windows, it is recommended to use **Git Bash**, which comes included as part of the **Git For Windows package**. It ca be obtained from the AstraZeneca Software Store by searching for “Git” and finding the icon called “Git 2.33” (or similar) and then clicking the “Request install” button. Alternatively, you can obtain it from the [**Git For Windows** package website](https://gitforwindows.org/).
```

Comment:
```suggestion
- On Windows, it is recommended to use **Git Bash**, which comes included as part of the **Git For Windows package**. It can be obtained from the AstraZeneca Software Store by searching for “Git” and finding the icon called “Git 2.33” (or similar) and then clicking the “Request install” button. Alternatively, you can obtain it from the [**Git For Windows** package website](https://gitforwindows.org/).
```